src/alto-ui/Form/InputNumber/InputNumber.js

```diff
@@ -71,14 +85,20 @@ class InputNumber extends React.Component {
   }
 
   parse(value) {
-    const { locale, precision, min, max } = this.props;
+    const { locale, precision, percent, min, max } = this.props;
     const res = parse(value, locale, precision, { min, max });
-    return Number.isNaN(res) ? '' : res;
+    if (Number.isNaN(res)) return '';
+
+    return percent ? res / 100 : res;
```

Comment:
this should be done in helper/number parse() function.

Reply:
moved :)